packages/@aws-cdk/aws-dynamodb-global/lib/aws-dynamodb-global.ts

```diff
@@ -0,0 +1,71 @@
+import dynamodb = require("@aws-cdk/aws-dynamodb");
+import cdk = require("@aws-cdk/cdk");
+import { LambdaGlobalDynamoDBMaker } from "./lambda-global-dynamodb";
+import { MultiDynamoDBStack } from "./multi-dynamodb-stack";
+
+export interface GlobalDynamoDBProps extends dynamodb.TableOptions {
+    /**
+     * Enforces a particular table name
+     * to share across Global DynamoDB tables
+     */
+    tableName: string;
+}
+
+/**
+ * Properties for the mutliple DynamoDB tables to mash together into a
+ * global table
+ */
+export interface GlobalTableProps extends cdk.StackProps {
```

Comment:
Sorry, I probably didn't explain myself too well (interfaces can extend multiple bases):

```ts
export interface GlobalTableProps extends cdk.StackProps, dynamodb.TableOptions {
  tableName: string;
  regions: string[];
}
```

Then, you don't need `GlobalDynamoDBProps` at all.

We like flat ;-)